### ISSUE TITLE
Issue with areas on same monitor model

### ISSUE THREAD

**Issue by JDoetsch85** [digitalcreations/MaxTo#508]
I just added a new monitor to my computer.  I have 2 LG ultrawide monitors of the same model and an old Acer monitor.  The new monitor is the 2nd LG Ultrawide.

Whenever I try to set the areas for the new ultrawide, it doesn't work.  Instead, that monitor copies the settings from the old ultrawide.  For instance, if I have the original LG set to 2 areas at 50% each, and I set the new LG set to 2 areas of 25% and 75%, when I save, both monitors will have 2 areas set at 50%.

If I set the old LG to 3 areas of 33% each and the new LG to 2 areas of 50% each, when I save, both monitors will be set to 3 areas of 33%.

It's almost as if MaxTo doesn't recognize that these are individual monitors and just treats them as one entity.

Any assistance would be appreciated

Cheers,
Joel

**Comment by vegardlarsen** [comment]
@JDoetsch85 Could this be a case where the two monitors have the same serial number (since they are of the same make and model)? Could you check the "Monitor details" when setting the regions?

Does anything change if you under "Monitor" in MaxTo's settings set identification to be "By properties" instead?

**Comment by JDoetsch85** [comment]
Hi @vegardlarsen 

It appears that none of my monitors is supplying a serial number to MaxTo, so it claims that it's falling back to properties.  I've confirmed that using properties instead of serial doesn't change the behaviour.

I tried the 3rd option for using windows numbering and that seems to have resolved the issue.  I'm not sure if you still want to keep this open.  Is it normal for the serial number to not get pulled into the program?

**Comment by vegardlarsen** [comment]
It is not uncommon that MaxTo cannot find the serial number, and in those cases it falls back to identifying the monitor by its properties. Properties include things such as which port the monitor is connected to, the monitor resolution and DPI, and so on.

There could be something wrong with how MaxTo detects the port the monitor is connected to. To help me debug this, I'll need a few things:

- Which version of MaxTo are you running?
- Can you get me the names of all the subfolders of `%AppData%\MaxTo\Monitor`? I don't need the contents, just the folder names.

**Comment by JDoetsch85** [comment]
Sorry, I've been on vacation the last week.  I'm thinking the issue is stemming from the fact that I'm using RDP to hook my laptop up to my desktop monitors.  In any case, the answer to your questions are that there is a folder with my laptop computer name, and under that folder is 2 more folders named "GenericPnP Monitor" and "Remote"

Version is 2.0.1

**Comment by vegardlarsen** [comment]
@JDoetsch85 It could be helpful to try upgrading to version 2.1.0-alpha.2; which has some fixes to some parts of how regions are loaded. But I don't think those fixes will do anything for your problem unfortunately.

**Comment by vegardlarsen** [comment]
@JDoetsch85 Can you follow up on if upgrading to 2.1.0-alpha.2 did help?

**Comment by JDoetsch85** [comment]
Somehow things have gotten worse.  Attempt to install new version failed.
It claimed it couldn't access a file in C:\Users\AppData.  Attempted to
turn off the program and install again, and no go.  At this point, I tried
using the MSI installer, and it didn't work.

Then I attempted to uninstall MaxTo and try a fresh install, and Windows
can't uninstall it (it claims it's missing a file to do so).

So long story short, I don't have a working version of MaxTo installed
anymore, I can't uninstall the ghost/stub version that's there, and I can't
install any new versions.

**Comment by vegardlarsen** [comment]
Could you possibly try this again with [2.1.0-alpha.4](https://maxto.net/release/alpha)? The best way to try a blank install is to:

1. Uninstall MaxTo
2. Reboot. (This step is important)
3. Install the new version.

**Comment by vegardlarsen** [comment]
Closing due to lack of feedback.
